.github/CODEOWNERS

```diff
@@ -183,6 +184,9 @@
 # Community Partners + Documentation
 /1e/manifest.json                        support@1e.com @DataDog/documentation @DataDog/marketplace-review
 /1e/README.md                            support@1e.com @DataDog/documentation @DataDog/marketplace-review
+/ably/*metadata.csv                      @ably support@ably.com @DataDog/documentation
+/ably/manifest.json                      @ably support@ably.com @DataDog/documentation
+/ably/README.md                          @ably support@ably.com @DataDog/documentation
```

Comment:
```suggestion
/ably/*metadata.csv                      @ably support@ably.com @DataDog/documentation @DataDog/marketplace-review
/ably/manifest.json                      @ably support@ably.com @DataDog/documentation @DataDog/marketplace-review
/ably/README.md                          @ably support@ably.com @DataDog/documentation @DataDog/marketplace-review
```

Reply:
Amended in 1443f6340151a93607a0added5f9867bf17a3c64